staging/src/k8s.io/cri-api/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- sig-node-approvers
+- dims
+reviewers:
+- sig-node-reviewers
+- dims
```

Comment:
nit: can we also add the `sig/node` label field here so that it's automatically applied?

Reply:
Done! thanks